impl/client/src/main/javascript/web/prompting/PromptPanel.js

```diff
@@ -34,1658 +33,1728 @@
  * @property {Function} onStateChanged Callback called if defined after state variables have been changed on the prompt panel or parameter definition
  * @property {?Function} onSubmit Callback called when the submit function executes, null if no callback is registered.
  */
-define(['cdf/lib/Base', 'cdf/Logger', 'dojo/number', 'dojo/i18n', 'common-ui/util/util', 'common-ui/util/GUIDHelper', './WidgetBuilder', 'cdf/Dashboard.Clean', './parameters/ParameterDefinitionDiffer', 'common-ui/jquery-clean', 'common-ui/underscore'],
-    function (Base, Logger, DojoNumber, i18n, Utils, GUIDHelper, WidgetBuilder, Dashboard, ParamDiff, $, _) {
-
-      // Add specific prompting message bundle
-      if (pentaho.common.Messages) {
-        pentaho.common.Messages.addUrlBundle('prompting',CONTEXT_PATH+'i18n?plugin=common-ui&name=resources/web/prompting/messages/messages');
+define(["cdf/lib/Base", "cdf/Logger", "dojo/number", "dojo/i18n", "common-ui/util/util", "common-ui/util/GUIDHelper", "./WidgetBuilder", "cdf/Dashboard.Clean", "./parameters/ParameterDefinitionDiffer", "common-ui/jquery-clean", "common-ui/underscore"],
+  function(Base, Logger, DojoNumber, i18n, Utils, GUIDHelper, WidgetBuilder, Dashboard, ParamDiff, $, _) {
+
+    // Add specific prompting message bundle
+    if(pentaho.common.Messages) {
+      pentaho.common.Messages.addUrlBundle("prompting", CONTEXT_PATH + "i18n?plugin=common-ui&name=resources/web/prompting/messages/messages");
+    }
+
+    var ERROR_MSG_POSITION_TOP = "top";
+    var ERROR_MSG_POSITION_BOTTOM = "bottom";
```

Comment:
IMO we need remove this not necessary logic related with changing error message position from PromptPanel and API.